collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -754,6 +757,16 @@ protected void onActivityResult(int requestCode, int resultCode, final Intent in
                 break;
             case RequestCodes.GEOSHAPE_CAPTURE:
                 String gshr = intent.getStringExtra(GEOSHAPE_RESULTS);
+                String[] gshrs = gshr.split(";");
+                // make sure the shape has at least three notes
```

Comment:
“notes” is meant to be nodes or something?

Reply:
Thank you for your careful inspection! I'm not a native english speaker so there will be some spelling mistakes sometimes. Will fix these soon !